test/expected-output-test/weights/input.ddl

```diff
@@ -0,0 +1,9 @@
+a?(k int).
+c?(k int).
+b(k int, p int, q text).
+
+a(x) :- b(x,y,_)
+weight = x + y.
+
+c(x) :- b(x,y,_)
+weight = "string".
```

Comment:
Since the parser supports multiple exprs for weight, how about adding that to this test as well?